app/views/component/detail/_address.html.haml

```diff
@@ -1,7 +1,7 @@
 - if location.address.present?
   %span{ itemscope: '', itemtype: 'http://schema.org/PostalAddress', itemprop: 'address' }
     %section.icon-text-block
-      %i.fa.fa-map-marker{ tabindex: "0", role: "img", aria: { label: "Location marker" } }
```

Comment:
This is not longer tab-able, the tabbing process skips all together the blue header box. Maybe we should keep those and allow people to tab through the headers and to be able to click by using only the keyboard on the service links.